app-text/dblatex/files/dblatex-0.3.7-path-logging.patch

```diff
@@ -0,0 +1,18 @@
+# from the wagnerflo overlay, see
+# https://bugs.gentoo.org/show_bug.cgi?id=129368
```

Comment:
We always try to get patches upstream, if they are suitable. If they're not, we try to improve them ;-)